From 6e2cefbf1c80a5d20655852e8ede17f71bda9e4e Mon Sep 17 00:00:00 2001
From: Quentin Casasnovas <quentin.casasnovas@vates.tech>
Date: Mon, 19 Oct 2026 14:27:15 +0200
Subject: [PATCH] Config changes.

Enable TX Queue support for better performance
Enable PTP (Precision Time Protocol) support (if enabled in kernel)
Enable RSS option for better performance
Enable automatic firmware loading, if optional firmware is available
Enable WOL magic packet by default
Disable EEE to improve compatibility

Signed-off-by: Quentin Casasnovas <quentin.casasnovas@vates.tech>
---
 src/Makefile | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index 9441926ad655..6d1e6083b145 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -38,14 +38,14 @@ CONFIG_DOWN_SPEED_100 = n
 CONFIG_ASPM = y
 ENABLE_S5WOL = y
 ENABLE_S5_KEEP_CURR_MAC = n
-ENABLE_EEE = y
-ENABLE_S0_MAGIC_PACKET = n
+ENABLE_EEE = n
+ENABLE_S0_MAGIC_PACKET = y
 ENABLE_TX_NO_CLOSE = y
-ENABLE_MULTIPLE_TX_QUEUE = n
-ENABLE_PTP_SUPPORT = n
-ENABLE_RSS_SUPPORT = n
+ENABLE_MULTIPLE_TX_QUEUE = y
+ENABLE_PTP_SUPPORT = y
+ENABLE_RSS_SUPPORT = y
 ENABLE_LIB_SUPPORT = n
-ENABLE_USE_FIRMWARE_FILE = n
+ENABLE_USE_FIRMWARE_FILE = y
 DISABLE_WOL_SUPPORT = n
 DISABLE_MULTI_MSIX_VECTOR = n
 ENABLE_DOUBLE_VLAN = n
